### PR TITLE
Fix Release URLs with new file name formats.

### PR DESCRIPTION
**Pull request by jdoss**
This PR fixes the links in the releases.

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -205,14 +205,13 @@ release:
...
-    - 📦 [step-cli_{{ .Version }}_amd64.deb](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ .Version }}_amd64.deb)
-    - 📦 [step-cli_{{ .Version }}_amd64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ .Version }}_amd64.rpm)
-    - 📦 [step-cli_{{ .Version }}_arm64.deb](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ .Version }}_arm64.deb)
-    - 📦 [step-cli_{{ .Version }}_arm64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ .Version }}_arm64.rpm)
+    - 📦 [step-cli_{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}_amd64.deb](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}_amd64.deb)
+    - 📦 [step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }.x86_64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.x86_64.rpm)
```
Comment (dopey) [inline .goreleaser.yml:212]:
```suggestion
    - 📦 [step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.x86_64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.x86_64.rpm)
```

File: .goreleaser.yml [hunk 2]
```diff
@@ -205,14 +205,13 @@ release:
...
+    - 📦 [step-cli_{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}_arm64.deb](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli_{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}_arm64.deb)
+    - 📦 [step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }.aarch64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.aarch64.rpm)
```
Comment (dopey) [inline .goreleaser.yml:214]:
```suggestion
    - 📦 [step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.aarch64.rpm](https://dl.smallstep.com/gh-release/cli/gh-release-header/{{ .Tag }}/step-cli-{{ replace .Version "-" "." }}-{{ .Var.packageRelease }}.aarch64.rpm)
```

**Review by dopey — APPROVED** [review]

**Comment by dbagarozza** [comment]
short question. 

Will the release urls remain as they are? That the url now contains the -1, the packageRelease variable. It wasnt like that before. :)

**Comment by jdoss** [comment]
Hey @dbagarozza we are slowly moving forward with releasing Yum and Apt repos and we are going to be following the conventional file name formats for RPM and Deb packages. We cannot guarantee we won't issue a release update that keeps the same version number but bumps the pacakgeRelease variable. I don't expect it to happen that often. 

If you are automating this on your end I would set the version you are grabbing to the full version + release (0.27.4-1 in this case).

**Pull request merged**